packages/thirdweb/src/chains/types.ts

```diff
@@ -26,6 +28,7 @@ export type ChainOptions = {
     increaseZeroByteCount?: boolean;
   };
   faucets?: Array<string>;
+  feeType?: FeeType;
```

Comment:
who will fill in this value though?

Reply:
when a user defines chain, it's good to be able to globally specify only to use legacy transaction
(this is the reason this came on my radar to begin with lol, I need a way to do this in engine)